crowdsec-docs/unversioned/cti_api/integration_securitycopilot.md

```diff
@@ -0,0 +1,53 @@
+---
+id: integration_securitycopilot
+title: Security Copilot Plugin
+---
+
+CrowdSec Intelligence Plugin for Microsoft Security Copilot allows you to get advanced insights on a malicious IP activity.  
+As part of the core plugins of Security copilot its setup and usage are very straight forward.
+
+This documentation will lead you through an easy setup and lead you through some example usage and prompts.
+
+## Configure the plugin
+
+### Prerequisite: retrieve your API Key
```

Comment:
We could just link to the getting started page of the CTI? or if you want to keep them here we can keep this